src/schemas/variable/VariableResponse.yaml

```diff
@@ -3,7 +3,12 @@ allOf:
   - type: object
     required:
       - scope
+      - key
     properties:
+      key:
+        type: string
+      value:
+        type: string
```

Comment:
Can we say `value` is `required` but can be `nullable`?

Reply:
we can try.